src/wallet/rpcwallet.cpp

```diff
@@ -1751,9 +1751,7 @@ UniValue rawshieldsendmany(const JSONRPCRequest& request)
         );
 
     CTransaction tx = CreateShieldedTransaction(request).getFinalTx();
-    UniValue tx_json(UniValue::VOBJ);
-    TxToUniv(tx, UINT256_ZERO, tx_json);
-    return tx_json;
+    return EncodeHexTx(tx);
 }
```

Comment:
we should probably do the same in `rawdelegatestake`

Reply:
updated